data/api/client-server/content-repo.yaml

```diff
@@ -644,14 +666,33 @@ paths:
                 enum:
                   - image/jpeg
                   - image/png
+                  - image/apng
+                  - image/gif
+                  - image/webp
           content:
             image/jpeg:
               schema:
                 # This is a workaround for us not being able to say the response is required.
                 description: "**Required.** The bytes for the thumbnail."
             image/png:
               schema:
-                description: "**Required.** The bytes for the thumbnail."
+                x-changedInMatrixVersion:
+                  "1.11": The PNG may be of the APNG variety if animation is supported and requested.
```

Comment:
For the record (no action required): these don't seem to show up in the rendered spec, though that's no reason not to add them.

Reply:
https://github.com/matrix-org/matrix-spec/issues/1774